cmd/argocd/commands/cluster.go

```diff
@@ -202,7 +204,7 @@ func NewClusterGetCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command
 // NewClusterRemoveCommand returns a new instance of an `argocd cluster list` command
 func NewClusterRemoveCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var command = &cobra.Command{
-		Use:   "rm",
+		Use:   "rm CLUSTER",
```

Comment:
minor - might be good to have an example

Reply:
I'm just fixing the basic usage. Cobra does support a separate section for showing examples, but I don't want to do it for just one sub command.